src/config.ts

```diff
@@ -194,7 +194,7 @@ function parseInternationalizationOptions(): InternationalizationOptions {
 			load: 'all',
 			lng: 'en-US',
 			fallbackLng: {
-				'es-419': ['es-ES'], // Latin America Spanish falls back to Spain Spanish
+				'es-419': ['es-ES', 'en-US'], // Latin America Spanish falls back to Spain Spanish
```

Comment:
Why is en-US here? In fact, I thought @vladfrangu updated discord-api-types recently to support es-419?

Reply:
Because if a key doesn’t exist in es-419, it’ll fallback to es-ES

And if it also doesn’t exist in es-ES, without en-US, i18next won’t be able to resolve the key.

This is merely so untranslated keys fallback correctly.